ci/testnet-automation.sh

```diff
@@ -39,16 +47,14 @@ launchTestnet() {
 
   echo --- start "$nodeCount" node test
   if [[ -n $USE_PREBUILT_CHANNEL_TARBALL ]]; then
-    net/net.sh start -f "cuda" -o noValidatorSanity -t "$CHANNEL"
+    net/net.sh start -f "cuda" -o noValidatorSanity -t "$CHANNEL" -c bench-tps=1="--tx_count 50000 --thread-batch-sleep-ms 1000"
```

Comment:
`-c 'bench-tps=1="--tx_count 50000 --thread-batch-sleep-ms 1000"'` oughta do it I'd think